FinSearchUnified/Tests/Bundle/SearchBundleFindologic/SearchQueryBuilderTest.php

```diff
@@ -34,6 +34,7 @@ class SearchQueryBuilderTest extends TestCase
      */
     protected function setUp()
     {
+        $this->markTestSkipped(sprintf('Obsolete test class: %s', get_class()));
```

Comment:
See my other comment regarding this.

Reply:
@TheKeymaster Should I also remove the associated classes too? Or just the test classes.

Reply:
@huzaifa-m if there are no longer used anywhere except from tests, they can be removed.